internal/admin/authorizedapps/save.go

```diff
@@ -27,16 +27,16 @@ import (
 	"github.com/google/exposure-notifications-server/internal/serverenv"
 )
 
-type saveController struct {
+type SaveController struct {
 	config *admin.Config
 	env    *serverenv.ServerEnv
 }
 
-func NewSave(c *admin.Config, env *serverenv.ServerEnv) *saveController {
-	return &saveController{config: c, env: env}
+func NewSave(c *admin.Config, env *serverenv.ServerEnv) *SaveController {
```

Comment:
I think the right fix is to have this return `admin.Controller` instead of `*SaveController`

My bad - all of the controller structs implement the admin.Controller interface and I missed the return type on the `New` methods.